test/stat/stat_test.go

```diff
@@ -46,13 +46,14 @@ var controllerDeployments = []string{"controller", "grafana", "prometheus", "web
 // requesting, and the test will pass.
 func TestCliStatForConduitNamespace(t *testing.T) {
 
+	// test `conduit stat deploy -n <namespace>`
```

Comment:
Rather than adding comments, I'd prefer to split this into two separate tests.

```
	t.Run("test conduit stat deploy", func(t *testing.T) {
		...
	})

	t.Run("test conduit stat namespace", func(t *testing.T) {
		...
	})
```